all/CMakeLists.txt

```diff
@@ -1,13 +1,14 @@
-cmake_minimum_required(VERSION 3.8)
+cmake_minimum_required(VERSION 3.14...3.20)
 
 project(exampleAll NONE)
 
-set(example_INCLUDE_WITHOUT_SYSTEM YES CACHE INTERNAL "")
+#XXX set(example_INCLUDE_WITHOUT_SYSTEM YES CACHE INTERNAL "")
```

Comment:
This is a superbuild CML for development. Why disable warnings during developments?